check50/__main__.py

```diff
@@ -285,15 +361,20 @@ def main():
                         metavar="FILE",
                         help=_("file to write output to"))
     parser.add_argument("-v", "--verbose",
+                        action="store_true",
+                        help=_("shows the full traceback of any errors and shows print statements written in checks"))
+    parser.add_argument("--log-level",
                         action="store",
-                        nargs="?",
-                        default="",
-                        const="info",
-                        choices=[attr for attr in dir(logging) if attr.isupper() and isinstance(getattr(logging, attr), int)],
+                        choices=list(LogLevel.__members__),
```

Comment:
Instead of accessing `__members__`, something like the following is probably cleaner:

```
[level.name.lower() for level in LogLevel]
```